corehq/apps/users/management/commands/add_multi_location_property.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.es import UserES, users as user_filters
+from corehq.apps.users.models import CouchUser
+from corehq.util.couch import iter_update, DocUpdate
+
+
+logger = logging.getLogger('user_migration')
+logger.setLevel('DEBUG')
+
+
+class Command(BaseCommand):
+    args = ""
+    help = ("")
+
+    def handle(self, *args, **options):
+        self.options = options
+        user_ids = self.get_user_ids()
```

Comment:
Quick plug for `with_progress_bar` - if you change this line to
```python
user_ids = with_progress_bar(self.get_user_ids())
```
you'll get a progress bar showing expected time remaining